src/app.js

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable quotes */
+/* eslint-disable curly */
+/* eslint-disable comma-dangle */
+/* eslint-disable operator-linebreak */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-use-before-define */
+/* eslint-disable new-cap */
+const listDisplayContainer = document.querySelector(
+  "[data-list-display-container]"
+);
+const listTitleElement = document.querySelector("[data-list-title]");
+const newListForm = document.querySelector("[data-new-list-form]");
+const newListInput = document.querySelector("[data-new-list-input]");
+const deleteListButton = document.querySelector("[data-delete-list-button]");
+
+const listCountElement = document.querySelector("[data-list-count]");
+const taskContainer = document.querySelector("[data-tasks]");
+const listsContainer = document.querySelector("[data-lists]");
+const newTaskForm = document.querySelector("[data-new-task-form]");
+
+const newPrioInput = document.querySelector("[data-new-priority-input]");
+const clearCompleteTasksButton = document.querySelector(
+  "[data-clear-complete-tasks-button]"
+);
+
+const newListTitle = document.querySelector("[data-new-title-input]");
+const newListDesc = document.querySelector("[data-new-desc-input]");
+const newListDate = document.querySelector("[data-new-date-input]");
+
+const LOCAL_STORAGE_LIST_KEY = "task.lists";
+const LOCAL_STORAGE_SELECTED_LIST_ID_KEY = "task.selectedListId";
+
+let lists = JSON.parse(localStorage.getItem(LOCAL_STORAGE_LIST_KEY)) || [];
+let selectedListId = localStorage.getItem(LOCAL_STORAGE_SELECTED_LIST_ID_KEY);
+
+const clearElement = (element) => {
+  while (element.firstChild) {
+    element.removeChild(element.firstChild);
+  }
+};
+
+const render = () => {
```

Comment:
- [ ] Under the my projects section, a default project should exist even after creating new projects.